requirements/test.txt

```diff
@@ -1,3 +1,4 @@
-django-nose==1.4.4
 mock==2.0.0
 coverage>=3.0
+pytest-django==3.0.0
+pytest-runner==2.9
```

Comment:
Moving forward, we should specify a minimum and maximum requirement. This reduces requirement conflicts within a project. Typically, it's good to specify a maximum requirement of the next major version. So here: `pytest-runner>=2.9,<3.0`.